src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -320,6 +307,19 @@ const formConfig = {
           uiSchema: vaEmployee.uiSchema,
           schema: vaEmployee.schema,
         },
+        waiveRetirementPay: {
+          title: 'Waiving Retirement Pay',
+          path: 'waive-retirement-pay',
```

Comment:
hey look at that - since the paths aren't chapter-dependent, moving them doesn't require any changes 🎉